src/OpenLoco/Industry.h

```diff
@@ -50,21 +50,21 @@ namespace OpenLoco
         BitSet<Limits::kMaxStations> var_E1;       // 0xE1 stations bit set
         StationId producedCargoStatsStation[2][4]; // 0x161
         uint8_t producedCargoStatsRating[2][4];    // 0x171
-        uint16_t var_179[2];
+        uint16_t productionRate[2];                // 0x179
         uint16_t var_17D[2];
         uint16_t var_181[2];
-        uint16_t var_185[2];
-        uint16_t produced_cargo_quantity[2]; // 0x189
-        uint16_t var_18D[3];
-        uint16_t required_cargo_quantity[3]; // 0x193
-        uint16_t var_199[3];
-        uint16_t var_19F[2];
-        uint16_t produced_cargo_max[2];        // 0x1A3 (produced_cargo_quantity / 8)
-        uint8_t produced_cargo_transported[2]; // 0x1A7 (%)
-        uint8_t historySize[2];                // 0x1A9 (<= 20 * 12)
-        uint8_t history_1[20 * 12];            // 0x1AB (20 years, 12 months)
-        uint8_t history_2[20 * 12];            // 0x29B
-        int32_t history_min_production[2];     // 0x38B
+        uint16_t producedCargoQuantityMonthlyTotal[2];           // 0x185
+        uint16_t producedCargoQuantityPreviousMonth[2];          // 0x189
+        uint16_t receivedCargoQuantityMonthlyTotal[3];           // 0x18D
+        uint16_t receivedCargoQuantityPreviousMonth[3];          // 0x193
+        uint16_t receivedCargoQuantityDailyTotal[3];             // 0x199
+        uint16_t producedCargoQuantityDeliveredMonthlyTotal[2];  // 0x19F
+        uint16_t producedCargoQuantityDeliveredPreviousMonth[2]; // 0x1A3
+        uint8_t producedCargoPercentTransportedPreviousMonth[2]; // 0x1A7 (%)
+        uint8_t producedCargoMonthlyHistorySize[2];              // 0x1A9 (<= 20 * 12)
+        uint8_t producedCargoMonthlyHistory1[20 * 12];           // 0x1AB (20 years, 12 months)
+        uint8_t producedCargoMonthlyHistory2[20 * 12];           // 0x29B
```

Comment:
Not too keen on the length of these properties, but I don't really have any real suggestions either. Perhaps drop `Cargo` and instead call them `productionQuantity` and `materialQuantity`? But that might be more ambiguous than what has already been introduced. Not too important right now.

Reply:
What i was thinking is carving this out into a separate struct. In an ideal world we would reorder it a bit but sadly not possible.